platform/linux-generic/odp_ml_quantize.c

```diff
@@ -62,6 +63,59 @@ void odp_ml_fp32_from_int8(float *fp32, const int8_t *i8, uint32_t num, float sc
 		fp32[i] = (float)(i8[i] - zerop) * scale;
 }
 
+void odp_ml_fp32_to_uint16(uint16_t *u16, const float *fp32, uint32_t num, float scale,
+			   uint16_t zerop)
+{
+	float fval;
+
+	_ODP_ASSERT(scale != 0);
```

Comment:
We have lately changed floating point comparison for other functions to avoid compiler warnings. This line should be changed to match those:
_ODP_ASSERT(scale < 0.0 || scale > 0.0);